lib/vrl/stdlib/src/parse_cef.rs

```diff
@@ -75,30 +122,87 @@ impl Function for ParseCef {
         arguments: ArgumentList,
     ) -> Compiled {
         let value = arguments.required("value");
-
-        Ok(ParseCefFn { value }.as_expr())
+        let translate_custom_fields = arguments
+            .optional("translate_custom_fields")
+            .unwrap_or_else(|| expr!(false));
```

Comment:
Small nitpick here, but I would probably keep `translate_custom_fields` as an `Option<Box<dyn Expression>>` in `ParseCefFn`, so in the default / common case `resolve` doesn't have to be called. (You can unwrap it as an actual `false` boolean below, rather than a VRL expression).

Reply:
Done.